charts/sftp/README.md

```diff
@@ -0,0 +1,100 @@
+# Sftp Helm Chart
+
+Helm Chart for setting up a sftp server
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Sftp Helm Chart](#sftp-helm-chart)
+  - [Prerequisites](#prerequisites)
+  - [Chart Details](#chart-details)
+    - [Verification](#verification)
+    - [Installation](#installation)
+    - [Example](#example)
+  - [Assigning a external IP to the SFTP](#assigning-a-external-ip-to-the-sftp)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Prerequisites
+
+* Kubernetes 1.8+ with beta APIs enabled
+
+## Chart Details
+
+This chart will do the following:
```

Comment:
Can we describe here that chart uses the atmoz/sftp:latest image to do the job?

Reply:
3fabf38